charts/helm-chart/kubernetes-dashboard/templates/_helpers.tpl

```diff
@@ -49,30 +49,16 @@ Create chart name and version as used by the chart label.
 Common labels
 */}}
 {{- define "kubernetes-dashboard.labels" -}}
-app.kubernetes.io/name: {{ include "kubernetes-dashboard.name" . }}
```

Comment:
We could adapt those templates to be closer to what Helm recommends in the newest versions:

```
{{/*
Common labels
*/}}
{{- define "monchart.labels" -}}
helm.sh/chart: {{ include "monchart.chart" . }}
{{ include "monchart.selectorLabels" . }}
{{- if .Chart.AppVersion }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
{{- end }}
app.kubernetes.io/managed-by: {{ .Release.Service }}
{{- end }}

{{/*
Selector labels
*/}}
{{- define "mychart.selectorLabels" -}}
app.kubernetes.io/name: {{ include "mychart.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
{{- end }}
```

It was not decided yet when the "old" template was written.

Would you like to keep your naming or should we use "full-upstream" mode given in my example and add `app.kubernetes.io/component` label to individual components?

Reply:
Yes, but the component label might differ from what I understand. It could be `web`, `database`, `api`, etc. I have tried to move it directly to resources and use proper values there. It might still need to be improved. I tried to follow this: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/